crds/objectstorage.k8s.io_bucketaccessrequests.yaml

```diff
@@ -36,20 +36,21 @@ spec:
             properties:
               bucketAccessClassName:
                 type: string
-              bucketAccessName:
+              bucketName:
                 type: string
               bucketRequestName:
                 type: string
               serviceAccountName:
                 type: string
             required:
             - bucketAccessClassName
-            - bucketRequestName
```

Comment:
BR is still needed in BAR.spec to handle sharing w/in the same namespace. (If I'm understanding this change correctly)

Reply:
In this line, it is only being removed from the REQUIRED list. Since it is either or with bucketName